src/frontend/src/pages/view-challenge/view-challenge.tsx

```diff
@@ -1,7 +1,104 @@
-import React from 'react';
+import React, { useState, ChangeEvent } from "react";
+import { Link } from "react-router-dom";
+import {
+    Button,
+    Box,
+    Grid,
+    Typography,
+    Container,
+    Paper,
+    Radio
+} from "@material-ui/core";
+
+import SubmitAnswerPopup from "./components/submit-answer-popup";
 
 export const ViewChallenge: React.FC = () => {
+    const [selectedValue, setSelectedValue] = useState("Def1");
+    const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+        setSelectedValue(event.target.value);
+    };
+
+    // TODO: When we need to implement the code
+    // knowing which option the user selected - https://reactjs.org/docs/hooks-state.html
     return (
-        <div>View Challenge Page</div>
-    )
+        <Container>
+            {/* // TOOD: This will be contained in the header */}
+            <Box p={3} display="flex">
+                <Link to="/home">
+                    <Button variant="contained">Home</Button>
+                </Link>
+            </Box>
+            <Grid
+                container
+                direction="column"
+                justify="center"
+                alignItems="center"
+                spacing={3}
+            >
+                <Grid item>
+                    <Typography variant="h4">WORD</Typography>
+                </Grid>
+                <Grid item>
+                    <Grid
+                        container
+                        direction="row"
+                        justify="space-evenly"
+                        alignItems="center"
+                        spacing={3}
+                    >
+                        <Grid item xs={10}>
```

Comment:
```suggestion
                        <Grid item>
```
I'm pretty sure the definitions were in a row for our low-fi prototypes.